nevergrad/optimization/optimizerlib.py

```diff
@@ -2275,6 +2277,35 @@ def __init__(
 NaiveIsoEMNA = EMNA().set_name("NaiveIsoEMNA", register=True)
 
 
+@registry.register
+class modcma(base.Optimizer):
+    def __init__(
+        self, parametrization: IntOrParameter, budget: tp.Optional[int] = None, num_workers: int = 1
+    ) -> None:
+        super().__init__(parametrization, budget=budget, num_workers=num_workers)
+        self.modcma = AskTellCMAES(
+            self.dimension, lambda_=max(num_workers, int(4 + 3 * np.log(self.dimension)))
+        )
+
+    def _internal_ask_candidate(self) -> p.Parameter:
+        data = self.modcma.ask()
+        ng_data = np.asarray(data, dtype=np.float_).flatten()
+        assert len(data) == self.dimension
+        assert len(ng_data) == self.dimension
```

Comment:
all those asserts should be useless and removed

Reply:
Removed.